### PR TITLE
SCMOD-7632: removed the logging of log4j and commons-logging

### PR DESCRIPTION
**Pull request by gaana-vishwnath**
JIRA: https://portal.digitalsafe.net/browse/SCMOD-7632
JENKINS: http://sou-jenkins2.hpeswlab.net/job/JobService/view/Developer/job/JobService~job-service~SCMOD-7632~CI/

### CODE REVIEW

**Comment by gaana-vishwnath** [comment]
This needs a change in the liquibase-installer version before the final merge

File: pom.xml [hunk 1]
```diff
@@ -77,7 +78,7 @@
             <dependency>
                 <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-liquibase-installer</artifactId>
-                <version>1.17.0-SNAPSHOT</version>
+                <version>1.17.0-SCMOD-7632-GV-SNAPSHOT</version>
```
Comment (dermot-hardy) [inline pom.xml:81]: Don't forget to revert before merging.

File: pom.xml [hunk 2]
```diff
@@ -274,19 +270,24 @@
...
+                <artifactId>slf4j-simple</artifactId>
+                <version>1.7.25</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-slf4j-impl</artifactId>
-                <version>2.0-rc2</version>
+                <groupId>com.github.cafapi</groupId>
+                <artifactId>caf-logging</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
```
Comment (dermot-hardy) [inline pom.xml]: This has now been released (and the coordinates have changed slightly).

File: worker-jobtracking-container/worker.sh [hunk 3]
```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 #
 
+log() {
+    echo "$@" | $(dirname "$0")/../scripts/caf-log-format.sh "${0##*/}"
```
Comment (dermot-hardy) [inline worker-jobtracking-container/worker.sh]: Second argument missing `$$`

File: job-service-scheduled-executor-container/worker.sh [hunk 4]
```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 #
 
+log() {
+    echo "$@" | $(dirname "$0")/../scripts/caf-log-format.sh "${0##*/}"
```
Comment (dermot-hardy) [inline job-service-scheduled-executor-container/worker.sh]: Second argument missing `$$`

File: job-service-postgres-container/pom.xml [hunk 5]
```diff
@@ -45,6 +45,16 @@
             <version>${project.version}</version>
             <classifier>jar-with-dependencies</classifier>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
```
Comment (dermot-hardy) [inline job-service-postgres-container/pom.xml]: Whitespace at end of line.

File: job-service-postgres-container/pom.xml [hunk 6]
```diff
@@ -45,6 +45,16 @@
...
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>            
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```
Comment (dermot-hardy) [inline job-service-postgres-container/pom.xml]: I can't see why these two are needed here.  Aren't they already in the uber-jar?

File: job-service-container/pom.xml [hunk 7]
```diff
@@ -151,6 +151,18 @@
             <artifactId>job-service-util</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- Logging -->
+        <dependency>
```
Comment (dermot-hardy) [inline job-service-container/pom.xml]: Whitespace at end of line

File: job-service-container/pom.xml [hunk 8]
```diff
@@ -151,6 +151,18 @@
...
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!-- End Logging -->
```
Comment (dermot-hardy) [inline job-service-container/pom.xml:166]: I don't think these two should be here, given that this is packaging war files and uber jars.  Did you add them for tests?  If so then they should be test scope.

File: job-service-container/pom.xml [hunk 9]
```diff
@@ -482,6 +494,14 @@
...
                                                 <useProjectArtifact>false</useProjectArtifact>
                                             </dependencySet>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <excludes>
+                                                    <exclude>com.github.jobservice:job-service-db</exclude>
+                                                </excludes>
+                                                <outputDirectory>/maven</outputDirectory>
+                                                <useTransitiveFiltering>true</useTransitiveFiltering>
+                                            </dependencySet>
```
Comment (dermot-hardy) [inline job-service-container/pom.xml]: What is the purpose of this new dependency set?

File: job-service-config/pom.xml [hunk 10]
```diff
@@ -35,11 +35,17 @@
...
-         <dependency>
+        <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
```
Comment (dermot-hardy) [inline job-service-config/pom.xml:46]: Exclusions should be in dependency management in the aggregator.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Comment by gaana-vishwnath** [comment]
closing this request as a different branches with the changes has been done and merged

**Pull request closed without merging**